examples/react-full/.testPages.ts

```diff
@@ -20,6 +20,11 @@ function testPages(
     expect(html).toContain("<h1>Welcome to <code>vite-plugin-ssr</code></h1>");
   });
 
+  test("link tags are correctly injected", async () => {
+    const html = await fetchHtml("/");
+    expect(html).toMatch(/<head>.*<title>.*<link.*<\/head>/s);
```

Comment:
Let's use `partRegex`, it's slightly more readable.

Reply:
`partRegex` does not seem to support flags? (The `/s` flag is necessary to match newlines.)

Reply:
You're right